src/jmcomic_ai/core.py

```
@@ -486,50 +700,89 @@ def __init__(self):
+                raw_path = plugin.decide_filepath(album, None, filename_rule, suffix, None, dir_rule_dict)
+                output_path = str(Path(raw_path).resolve())
+
+            self.logger.info(f"Post-process '{process_type}' finished. Output: {output_path}")
+            return {
+                "status": "success",
+                "process_type": process_type,
+                "album_id": album_id,
+                "output_path": output_path,
+                "is_directory": is_directory,
+                "message": f"Post-process '{process_type}' completed successfully."
+            }
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)